examples/src/main/java/com/crawljax/examples/RemoteExample.java

```diff
@@ -0,0 +1,51 @@
+package com.crawljax.examples;
+
+import com.crawljax.core.CrawljaxRunner;
+import com.crawljax.core.configuration.BrowserConfiguration;
+import com.crawljax.core.configuration.CrawlRules.FormFillMode;
+import com.crawljax.core.configuration.CrawljaxConfiguration;
+import com.crawljax.core.configuration.CrawljaxConfiguration.CrawljaxConfigurationBuilder;
+import com.crawljax.plugins.crawloverview.CrawlOverview;
+import com.crawljax.plugins.testcasegenerator.TestConfiguration;
+import com.crawljax.plugins.testcasegenerator.TestConfiguration.StateEquivalenceAssertionMode;
+import com.crawljax.plugins.testcasegenerator.TestSuiteGenerator;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.remote.Browser;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+public class RemoteExample {
+
+    private static final long WAIT_TIME_AFTER_EVENT = 200;
+    private static final long WAIT_TIME_AFTER_RELOAD = 200;
+
+    /**
+     * Run this method to start the crawl.
+     */
+    public static void main(String[] args) {
+        String remoteUrl = "http://localhost:4444/wd/hub";
```

Comment:
Can you add comments on how to start this service locally?

Reply:
I added comments with links to help get started.